public/app/themes/justice/style.css

```diff
@@ -4,7 +4,7 @@ Theme URI: https://www.justice.gov.uk/
 Author: Ministry of Justice | Justice Digital | Central Digital Product Team
 Author URI: https://www.gov.uk/government/organisations/ministry-of-justice
 Description: A theme designed to display public-facing, procedural information for the Ministry of Justice, UK
-Version: 1.2
+Version: 10.1.2
```

Comment:
Why did we do this?

Reply:
It doesn't look pretty, but will mean the site looks less interesting during automated scans.

Reply:
Hi @wilson1000 I'll resolve these 2 conversations so that we can get this merged, but I'm happy to revisit if you think we need to.